bindings/gumjs/gumcmodule.c

```diff
@@ -28,11 +28,20 @@ struct _GumCModule
   GumCModuleFinalizeFunc finalize;
 };
 
+enum _GumCModuleHeaderKind
+{
+  GUM_CMODULE_HEADER_TCC,
+  GUM_CMODULE_HEADER_FRIDA
+};
+
+typedef enum _GumCModuleHeaderKind GumCModuleHeaderKind;
```

Comment:
Let's move this typedef up to the other typedefs [*], and use `guint` as the type there.

[*] Their order should match the order of the struct/enum definitions below.

Reply:
Ok.